install.ps1

```diff
@@ -0,0 +1,78 @@
+# Copyright 2022 The NATS Authors
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+$ErrorActionPreference = 'Stop'
+
+$Version = ""
+if ($args.Length -eq 1) {
+    $Version = $args.Get(0)
+}
+
+$NscInstall = $env:NSC_INSTALL
+$BinDir = if ($NscInstall) {
+    "$NscInstall\bin"
+} else {
+    "$Home\.nats\bin"

```

Comment:
The install location should be `$HOME/.local/bin`

Reply:
1. By windows convention, binaries belong in `C:\Program Files\<appname>\` (or sometimes `C:\Program Files\<appname>\bin\`) when being made available for all users. When being added only for the current user, binaries go under `$Home\AppData\Local\<appname>\`

2. `.local` means nothing to Windows users and is confusing. Again, the Windows equivalent is `$Home\AppData\Local\`

3. `$Home\.nats\bin\` is a compromise. As it is common to have settings and configuration for an app under `$Home\.<appname>\`, there will be some anchor for Windows users. Adding `bin` was what Deno did and it made sense to me to separate binaries and config. 

4. The CLI uses `$Home\.config\nats\`. The `.config` location is also a Unix concept and should not have been used for Windows. `$Home\.nats\` would have been better.

We don't force clients to be exactly identical, we use and expect language specific paradigms. The same thing should apply here and we should use OS specific paradigms.